src/repositories/groups-repository.js

```diff
@@ -50,7 +50,7 @@ module.exports = function createGroupsRepository(sequelize) {
   }
 
   async function add(form) {

```

Comment:
Form is not a proper name, and is misleading, use the lowercase specialty or data.